src/components/tools/Charts/LineArea/Chart.svelte

```diff
@@ -79,6 +82,18 @@
       );
     }
 
+    // Derive a tolerance value using precision specified by user, defaults to 1
+    // Precision represents the number of digits past the decimal point
+    const tolerance = yAxis.precision
```

Comment:
I'm not sure I understand this part; the value of `tolerance` will always be a string like "1.0" or "1.00" (when `precision` is 2). You're also passing this string to the `almostEqual` function when it looks like it expects a decimal value for the 3rd argument, not a string.

Reply:
Right, it should be a float value and not a string value. But yes, the default float value will be `1.0`

Reply:
So is the idea to change it to another float like 2.0, 1.5, etc?